packages/sanity/src/structure/panes/document/documentPanel/DocumentPanel.tsx

```diff
@@ -188,7 +206,8 @@ export const DocumentPanel = function DocumentPanel(props: DocumentPanelProps) {
       displayed?._id &&
       getVersionFromId(displayed._id) !== selectedReleaseId &&
       ready &&
-      !isPinnedDraftOrPublish
+      !isPinnedDraftOrPublish &&
+      isNewDocument(editState) === false
```

Comment:
this is a super nitpick, I saw this somewhere else too, but is the `=== false `necessarily in this?

wouldn't 

```suggestion
      !isNewDocument(editState)
```

be enough?

I understand that it might be because it returns undefined instead but it feels like it'll be prone of error? When I looked at it because my instinct upon looking at the isNewDocument (as someone that didn't implement it) was that if it gave me a linting error at trying the suggestion above I'd likely try to `Boolean(isNewDocument(editState))` which then would give me the wrong expectation (Because undefined would return false but technically it's only still loading

this might be an issue of like "maybe you should read the comments attached to the method" but I know developers because I am one 😂